### PR TITLE
Display the unit icons as circles in the machine view

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3984/
Test PASSed.

**Comment by hatched** [comment]
:+1: :ok: :shipit: 
It's unfortunate that every icon has that little black line at the top of the circle now. Hopefully we can update those icons soon.

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

File: jujugui/static/gui/src/app/components/machine-view/machine/_machine.scss [hunk 1]
```diff
@@ -95,6 +89,14 @@
         margin-right: 10px;
         padding: 2px;
         border: 1px solid $canvas-background;
-        border-radius: 6px;
+        border-radius: 50%;
+
+        img {
```
Comment (anthonydillon) [inline jujugui/static/gui/src/app/components/machine-view/machine/_machine.scss]: We shouldn't target the element here. As a component we shouldn't only target classes as the element may change but we don't want the styling to be lost. For example this element maybe be changed to picture in the future which with require two edits.

**Pull request merged**